hw-vendor/marvell/bindata/daemonset.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: mrvl-comm-channel
+  # namespace: {{.Namespace}}
```

Comment:
The VSP daemonset should be rendered by the controller. Did Marvell have challenges working with the controller code?